.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,20 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets:
+      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+      BALENA_API_KEY: ${{ secrets.BALENA_API_KEY }}
+    with:
+      protect_branch: false
```

Comment:
This should be removed before merging

Reply:
How does this interact with the deploy-to-balena action setup here https://github.com/balena-os/balena-supervisor/blob/master/.github/workflows/balena-ci.yml?

Reply:
Maybe `balena-ci.yml` isn't needed anymore because Flowzone [pushes draft releases to balena](https://github.com/product-os/flowzone#balena) if `balena.yml` is present in project root @pipex ? I'm asking in [JF](https://jel.ly.fish/5e1dc3c9-fc6e-44fe-b60a-765e9fe21f38?event=1da613db-ea58-4d53-9f95-809900f10ea2) as well to see if there are any existing repos we can look at as an example.

Reply:
I would expect that flowzone could also replace the deploy-to-balena config @cywang117, but the DTB worflow configures the fleet names and also the push to staging, I don't know if all of that is deduced by just the contract

Reply:
May need to sleep this PR until https://github.com/product-os/flowzone/issues/198 is resolved. From the (internal) JF thread I linked above, Flowzone would need to support setting the `environment` parameter as in the balena-ci.yml config, and subsequently would need a scalable way of figuring out the balena API secret env var.

Reply:
Waiting on admin access to repo to set FLOWZONE_TOKEN, so we can test the Flowzone workflow with versioning.

Reply:
balena-ci.yml was removed with latest config and the draft releases on balena cloud & staging are there and have the correct version.